programmers/비밀지도.js

```diff
@@ -0,0 +1,32 @@
+function solution(n, arr1, arr2) {
+    const passwords = [];
+    const results = [];
+    
+    let sum = 0;
+    let str = '';
+
+    for (let i = 0; i < arr1.length; i++) {
+        sum = arr1[i] | arr2[i];
+        sum = sum.toString(2);
+        passwords.push(sum)
+    }
```

Comment:
sum 값을 대입해 주는게 두번 반복하기보다 이렇게 한 줄로 작성해 주면 더 간결하게 작성 가능해요
```suggestion
    for (let i = 0; i < arr1.length; i++) {
        sum = (arr1[i] | arr2[i]).toString(2);
        passwords.push(sum);
    }
```